src/pages/new/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function New() {
```

Comment:
What's this for?

Reply:
This was a test component when i was trying to make changes, will delete it.